huxley/core/models.py

```diff
@@ -394,6 +394,11 @@ class Delegate(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     summary = models.TextField(default='', blank=True, null=True)
 
+    friday_attendance = models.BooleanField(default=False)
```

Comment:
It's per assignment. We are (at least for now) just planning on duplicating the information we get per assignment to each delegate assigned to that assignment